_sass/blocks/revenues/_revenues_subpage_diving_graphics.scss

```diff
@@ -196,113 +191,117 @@
         z-index: 2;
       }
     }
-  }
-
-  &.coal{
 
-    background: $coal-primary-color;
+    background: $offshore-renewables-primary-color;
```

Comment:
Expected item on line 195 to appear before line 167. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets